flink-python/src/main/java/org/apache/flink/python/env/beam/ProcessPythonEnvironmentManager.java

```diff
@@ -347,10 +350,13 @@ private void constructArchivesDirectory(Map<String, String> env) throws IOExcept
         }
     }
 
-    private void constructRequirementsDirectory(Map<String, String> env) throws IOException {
+    private void constructRequirementsDirectory(Map<String, String> env, String baseDirectory)
```

Comment:
baseDirectory could be got from `getBaseDirectory` directly and so this change could be reverted